Documentation/devel/stage1-implementors-guide.md

```diff
@@ -16,36 +16,47 @@ This separation of concerns is reflected in the file-system and layout of the co
 1. Stage 1: "stage1.aci", made available at "/var/lib/rkt/pods/$uuid/stage1" by `rkt run`.
 2. Stage 2: "$app.aci", made available at "/var/lib/rkt/pods/$uuid/stage1/rootfs/opt/stage2/$appname" by `rkt run`, where $appname is the name of the app in the pod manifest.
 
-The stage 1 implementation is what creates the execution environment for the contained applications.  This occurs via entrypoints from stage 0 on behalf of `rkt run` and `rkt enter`.  These entrypoints are nothing more than executable programs located via Annotations from within the stage 1 ACI manifest, and executed from within the stage 1 of a given pod at "/var/lib/rkt/pods/$uuid/stage1/rootfs".
+The stage 1 implementation is what creates the execution environment for the contained applications.
+This occurs via entrypoints from stage 0 on behalf of `rkt run` and `rkt enter`.
+These entrypoints are nothing more than executable programs located via Annotations from within the stage 1 ACI manifest, and executed from within the stage 1 of a given pod at "/var/lib/rkt/pods/$uuid/stage1/rootfs".
```

Comment:
These entrypoints are executable programs located via Annotations from within the stage 1 ACI manifest, and executed from within the stage 1 of a given pod at `/var/lib/rkt/pods/$uuid/stage1/rootfs`.